pkg/proto/runtime/v1/dapr_additional.go

```diff
@@ -62,3 +67,87 @@ func (x *SubtleVerifyAlpha1Request) SetComponentName(name string) {
 		x.ComponentName = name
 	}
 }
+
+// CryptoRequests is an interface for EncryptAlpha1Request and DecryptAlpha1Request.
+type CryptoRequests interface {
+	proto.Message
+
+	// SetPayload sets the payload.
+	SetPayload(payload *commonv1pb.StreamPayload)
+	// GetPayload returns the payload.
+	GetPayload() *commonv1pb.StreamPayload
+	// Reset the object.
+	Reset()
+	// SetOptions sets the Options property.
+	SetOptions(opts proto.Message)
+	// HasOptions returns true if the Options property is not empty.
+	HasOptions() bool
+}
+
+func (x *EncryptAlpha1Request) SetPayload(payload *commonv1pb.StreamPayload) {
```

Comment:
Would cause compile-time issues if fields are renamed. However renaming a field in protobuf is a breaking change so that should not happen. Adding new fields is ok.